external_search.py

```diff
@@ -1775,7 +1774,7 @@ def universal_nested_filters(cls):
         not_suppressed = F('term', **{'licensepools.suppressed' : False})
         nested_filters['licensepools'].append(not_suppressed)
 
-        owns_licenses = F('term', **{'licensepools.owned' : True})
+        owns_licenses = F('term', **{'licensepools.licensed' : True})
```

Comment:
While debugging this, I found the 'owned' language confusing -- open access books are not 'owned' -- so I renamed the field to 'licensed'.